MAINTAINERS

```diff
@@ -0,0 +1,17 @@
+# Official list of OpenEBS Maintainers.
+#
+# Names added to this file should be in the following format:
+#     Individual's name,@githubhandle, Company Name
+#
+# Please keep the below list sorted in ascending order.
+#
+#Maintainers
+"Kiran Mova",@kmova,MayaData
+"Jeffry Molanus",@gila,MayaData
```

Comment:
The names are not sorted in ascending order.

Reply:
thank you! fixed.